listenbrainz/webserver/templates/index/current-status.html

```diff
@@ -28,6 +28,14 @@ <h3>ListenBrainz Stats</h3>
             <td>{{ listen_count }}</td>
           </tr>
         {% endif %}
+        {% if listen_counts_per_day %}
```

Comment:
The screenshot of current status shows today and yesterday's listens, but that isn't reflected in the PR here. Missing commit?

Reply:
`listen_counts_per_day` is a list, and we loop through it later in the template.

Reply:
Oh heh, missed the loop.